stream_alert/__init__.py

```diff
@@ -1 +1,107 @@
-__version__ = '1.1.0'
\ No newline at end of file
+import json
+import logging
+import os
+
+from stream_alert.config import load_config, load_env
+from stream_alert.classifier import StreamPayload, StreamClassifier
+from stream_alert.pre_parsers import StreamPreParsers
+from stream_alert.rules_engine import StreamRules
+from stream_alert.sink import StreamSink
+
+logging.basicConfig()
+logger = logging.getLogger('StreamAlert')
+logger.setLevel(logging.INFO)
+
+__version__ = '1.2.0'
+
+class StreamAlert(object):
```

Comment:
I don't think we should put a full class in `__init__.py`, because then it can't be imported directly. `__init__.py` should be used for [structuring a package](https://docs.python.org/2.7/tutorial/modules.html#packages), not defining classes.